weather_command/_builder.py

```diff
@@ -348,7 +348,7 @@ def _format_date_time(
 
 
 def _format_precip(precip_amount: float | None, units: str) -> str:
-    if not precip_amount:
+    if not precip_amount or _mm_to_in(precip_amount) < 0.01:
         return "0"
```

Comment:
`_mm_to_in` is rounding to 2 decimal places so you don't need to worry about this being less than 0.01. Also the return should be "0.00"

```py
def _format_precip(precip_amount: float | None, units: str) -> str:
    if not precip_amount:
        return "0.00"
```

Reply:
Sorry I must of misread the issue.